src/dev/build/args.ts

```diff
@@ -94,7 +91,6 @@ export function readCliArgs(argv: string[]) {
   const buildOptions: BuildOptions = {
     isRelease: Boolean(flags.release),
     versionQualifier: flags['version-qualifier'],
-    buildOssDist: flags.oss !== false,
     buildDefaultDist: !flags.oss,
```

Comment:
While you're in here I feel like we should remove this property from `BuildOptions` too, it's always true now that we're building the "default" version of the distributable.